src/library/Cardano/CoinSelection/Migration.hs

```diff
@@ -8,11 +8,10 @@
 -- Copyright: © 2018-2020 IOHK
 -- License: Apache-2.0
 --
--- This module contains an algorithm to select coins for migration from legacy
--- wallets to newer wallets.
+-- This module contains an algorithm to select coins for emptying a wallet.
 --
--- We want users to be able to migrate their funds from a legacy random wallet
--- to a new sequential wallet. To do this, we have to move funds from a wallet
+-- A typical use-case could be users who want to migrate their funds from an old
+-- wallet to a new wallet. To do this, we have to move funds from a wallet
 -- to another by making transactions. Funds are ultimately a sum of many coins
 -- (a.k.a UTxOs). In a transaction, we can select a few coins, and send them to
 -- addresses, effectively creating new coins / UTxOs doing this.
```

Comment:
Perhaps the last couple of lines are superfluous, as they pertain to transactions in general, rather than specifically to migration. (I think we already describe transactions elsewhere.)

How about:
```suggestion
-- to another by making a /series/ of transactions.
```